oci/layer/tar_extract.go

```diff
@@ -136,7 +144,11 @@ func (te *TarExtractor) restoreMetadata(path string, hdr *tar.Header) error {
 		if errors.Cause(err) != unix.ENOTSUP {
 			return errors.Wrapf(err, "clear xattr metadata: %s", path)
 		}
-		log.Warnf("xattr{%s} ignoring ENOTSUP on clearxattrs", path)
+		if !te.enotsupWarned {
+			log.Warnf("xattr{%s} ignoring ENOTSUP on clearxattrs", path)
+			log.Warnf("xattr{%s} destination filesystem does not support xattrs, further warnings will be suppressed", path)
+			te.enotsupWarned = true
```

Comment:
This patch does make sense, my only comment is that this behaviour should be disabled if the logging level is debug.